lib/ansible/modules/cloud/amazon/elasticache_snapshot.py

```diff
@@ -174,7 +174,8 @@ def delete(module, connection, name):
             response = {}
             changed = False
         elif e.response['Error']['Code'] == "InvalidSnapshotState":
-            module.fail_json(msg="Error: InvalidSnapshotState. The snapshot is not in an available state or failed state to allow deletion. You may need to wait a few minutes.")
+            module.fail_json(msg="Error: InvalidSnapshotState. The snapshot is not in an available state or failed state to allow deletion."
```

Comment:
This just joins the two strings, so you need a trailing space on this line so it doesn't come out as `...allow deletion.You may need...`